src/Symfony/Component/JsonPath/Functions/JsonPathFunctionArgumentTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonPath\Functions;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+trait JsonPathFunctionArgumentTrait
+{
+    public static function assertArgumentsCount(array $args, int $expectedCount): void
```

Comment:
why making the public static methods if this is in a trait ?

Either we should have public static utilities in a dedicated class expected to be called statically, or we should have a trait defining private instance methods (so that using the trait does not expend the public API of the class implementing a function)